doma-processor/src/main/java/org/seasar/doma/internal/apt/meta/entity/EntityMetaScope.java

```diff
@@ -0,0 +1,20 @@
+package org.seasar.doma.internal.apt.meta.entity;
+
+import org.seasar.doma.internal.apt.annot.ScopeClass;
+
+public class EntityMetaScope {
```

Comment:
I feel `ScopeMeta` is more appropriate.

Reply:
Deleted because it is no longer needed. https://github.com/domaframework/doma/pull/643/commits/30a461e1b38aa72b1427a4f6ded996e4b4395dc9